htdocs/search.php

```diff
@@ -1108,17 +1120,11 @@
         }
 
         // additional options
-        if (!isset($options['f_userowner'])) {
-            $options['f_userowner'] = '0';
-        }
-        if ($options['f_userowner'] != 0) {  // Ocprop
+        if ($options['f_userowner'] === 1) {  // Ocprop
```

Comment:
u should migrate to Symfony Request where possible. this strict type is at risk as an int is not guaranteed see https://www.w3schools.com/php/filter_validate_boolean.asp

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/ParameterBag.php#L185